acme4j-client/src/main/java/org/shredzone/acme4j/Certificate.java

```diff
@@ -115,13 +115,29 @@ public X509Certificate getCertificate() {
     }
 
     /**
-     * Writes the certificate to the given writer. It is written in PEM format, with the
-     * end-entity cert coming first, followed by the intermediate ceritificates.
+     * Writes the certificate and the intermediate certificates to the given writer.
+     * They are written in PEM format, with the end-entity cert coming first, followed
+     * by the intermediate certificates.
+     * This method just calls writeCertificateAndChain.
+     * 
+     * @see #writeCertificateAndChain
      *
      * @param out
      *            {@link Writer} to write to. The writer is not closed after use.
      */
     public void writeCertificate(@WillNotClose Writer out) throws IOException {
+        writeCertificateAndChain(out);
```

Comment:
I agree that `writeCertificateAndChain()` is a better name for the method. However there are now two methods doing exactly the same thing. I prefer to keep `writeCertificate()` like it was before (for downward compatibility), and remove `writeCertificateAndChain()`.